### ISSUE TITLE
Display price information in deploy dialog

### ISSUE THREAD

**Issue by st** [slipstream/SlipStreamUI#636]
Price information (from PRS) are displayed after each connector.
e.g
`exoscale (USD 123.00)`

### UI side

- [x] fix: do not accumulate decorations
- [x] first version for component dialog 
- [x] version for application dialog (multiple components)
- [x] fix: when server becomes unavailable, reset previous computation
- [x] when price is negative, `not available` is displayed instead
- [x] ~~when connector (from original user list) is not returned, then it is greyed out in dropdown~~ (later when filtering active)
- [x] order connectors in dropdown according to price (cheaper first)
- [x] check that when PRS "broken" there is no impact
- [x] multiplicity
- [x] nice-to-have: update app price when connectors chosen at node levels (hybrid deployment)
- [ ] display warning text in dialog : "this is an estimation + no network cost taken into account"
- [x] ~~allow duration selection for price estimation (hour, day, week)~~ FIXED hour for now
- [x] display `/ hour` in dialog
- [x] Use hour instead of minutes for fixed period
- [x] configure obsolete period for caching

### server side
- [ ] support prices for amazon, ultimum, exoscale, orange (cloudwatt) 
- [x] adapt scraping service
- [x] add ranking information
- [ ] configure scraping service (see phantom.js version)
- [x] dependency between service-offer and printing lib : how to mock for unit tests?
- [x] call service-offer to get prices
- [x] transmit vm-sizes (fix structure!)
- [x] replace hard-coded values by configuration (service-offer end-point)
- [x] check performance
- [x] ~~filter using CIMI prices from service-offer~~

### infra
- [x] clean up boot dependencies
- [ ] get rid of http-kit

### demo
- [ ] check after deployment env variables are defined 

Part of https://github.com/SixSq/tasklist/issues/784

**Comment by st** [comment]
Available clouds : GCE, exoscale, ultimum.

Proposed plan of action is to:
*  hard-code results of scraping inside Pricing-lib (probably use Json files similar to examples at the end of `sixsq.slipstream.pricing.lib.pricing`)
* work on scraping and pricing-lib tuning can be done in parallel
